drivers/hmis/lib/form_data/static/project_config.json

```diff
@@ -1,10 +1,15 @@
 {
   "item": [
     {
-      "type": "INTEGER",
-      "link_id": "lengthOfAbsenceDays",
-      "text": "Length of Absence (Days)",
-      "mapping": { "field_name": "lengthOfAbsenceDays" }
+      "type": "CHOICE",
+      "link_id": "configType",
+      "text": "Config Type",
+      "pick_list_reference": "ProjectConfigType",
+      "mapping": {
```

Comment:
I noticed that, if there is an existing auto_exit_config row in the `hmis_form_definitions` table, then the frontend /admins/form interface will fail with a message saying that the form role is not allowed. Should we write another migration to mitigate this?